libexec/python/docker/Makefile.am

```diff
@@ -1,6 +1,6 @@
 scriptlibexecdir = $(libexecdir)/singularity/python/docker
 
-dist_scriptlibexec_SCRIPTS = api.py __init__.py converter.py
+dist_scriptlibexec_DATA = api.py __init__.py converter.py
 
```

Comment:
@gmkurtzer does this fit with your standard for the Makefiles? What is the distinction betwen DATA and SCRIPTS?